lib/src/main/java/com/segment/analytics/kotlin/destinations/consent/Constants.kt

```diff
@@ -0,0 +1,14 @@
+package com.segment.analytics.kotlin.destinations.consent
+
+class Constants {
```

Comment:
maybe make it an object? so we don't need to define a companion object again. 
```suggestion
object Constants {
```